libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -1395,7 +1408,13 @@ - (void)oauthCoordinator:(SFOAuthCoordinator *)coordinator displayAlertMessage:(
                                                           }];
     
     [alert addAction:defaultAction];
-    [[SFRootViewManager sharedManager] pushViewController:alert];
+    __weak typeof(self) weakSelf = self;
+    SFSDKWindowManager.sharedManager.authWindow.viewController = [self blankViewController];
+    [SFSDKWindowManager.sharedManager.authWindow enable:NO withCompletion:^{
+            [SFSDKWindowManager.sharedManager.authWindow.viewController presentViewController:weakSelf.statusAlert animated:NO completion:^{
```

Comment:
It looks like this completion block is empty. Can we replace it with `nil` instead?